src/index.js

```diff
@@ -1,13 +1,132 @@
+// @flow
+
+import { get } from 'lodash';
+
+export type MongoOptionsType = {
+  collectionName: string,
+  timestamps: {
+    createdAt: string,
+    updatedAt: string,
+  }
+};
+
+export type MongoUserObjectType = {
+  username?: string,
+  profile?: Object,
+  services: {
+    password?: {
+      bcrypt: string,
+    },
+  },
+  emails?: [{
+    address: string,
+    verified: boolean,
+  }],
+};
+
+// TODO Will import from @accounts/account once it's published on npm
+export type UserObjectType = {
+  username: ?string,
+  email: ?string,
+  id: ?string,
+  profile: ?Object,
+  password: ?string,
+};
+
 class Mongo {
-  constructor(options) {
+  options: MongoOptionsType;
+  // TODO definition for mongodb connection object
+  db: any;
+  collection: any;
+
+  constructor(db: any, options: MongoOptionsType) {
```

Comment:
Take a look here https://flowtype.org/docs/third-party.html to allow mongo driver typings